### PR TITLE
Add explicit error return

### PR DESCRIPTION
**Pull request by mo1ein**

### CODE REVIEW

File: github/repos_contents.go [hunk 1]
```diff
@@ -317,5 +317,8 @@ func (s *RepositoriesService) GetArchiveLink(ctx context.Context, owner, repo st
 	}
 
 	parsedURL, err := url.Parse(resp.Header.Get("Location"))
-	return parsedURL, newResponse(resp), err
+	if err != nil {
+		return nil, newResponse(resp), err
+	}
+	return parsedURL, newResponse(resp), nil
```
Comment (gmlewis) [inline github/repos_contents.go:323]: I know this is another thing that is inconsistent across this repo, but while I'm thinking about it, let's go ahead and add a blank line between the error handling and the final return, please.

```suggestion
	}

	return parsedURL, newResponse(resp), nil
```

**Comment by gmlewis** [comment]
Note that I'm going to change the title of this PR because it is technically not an "unhandled error"... it is simply an error that is not explicitly tested for.

File: github/repos_contents.go (hunk 1 above)
Comment (mo1ein) [reply github/repos_contents.go]: - [x] Done.

**Review by gmlewis — APPROVED** [review]
Thank you, @mo1ein !
LGTM.
Merging.

**Pull request merged**